mirror/modules/sink/http.go

```diff
@@ -42,9 +42,10 @@ func init() {
 }
 
 type HTTPConfig struct {
-	TargetURL *expr.StringExpr `json:"target_url,omitempty"`
-	Timeout   string           `json:"timeout,omitempty"`
-	Parallel  int              `json:"parallel"`
+	FollowRedirect string           `json:"follow_redirect,omitempty"`
```

Comment:
Can be a bool. This way you won't have to parse the string, and you can use true/false in your json

Reply:
Indeed, much nicer.